draft-fossati-cose-profiles.md

```diff
@@ -80,9 +82,29 @@ A COSE profile:
     * using an OID {{-oid}}, URI {{-uri}} or CRI {{-cri}}
     * using a UUID {{-uuid}}
   * The chosen value SHOULD be appropriate for the intended usage scope (e.g., a small value when used in constrained node environments)
-* MAY define its own COSE tag
+* MAY define its own CBOR tag
```

Comment:
we should probably get even more specific here (in later iterations)

Reply:
would you mind raising an issue to track future actions?